arwen/elrondapi/bigintOps.go

```diff
@@ -132,6 +132,9 @@ func bigIntgetArgument(context unsafe.Pointer, id int32, destination int32) {
 
 	value := hostContext.GetOne(destination)
 	value.Set(args[id])
+
+	gasToUse := hostContext.GasSchedule().BigIntAPICost.BigIntGetArgument
+	hostContext.UseGas(gasToUse)
```

Comment:
`bigIntgetArgument()` should consume gas even if the received argument `id` is invalid.

Reply:
done